### PR TITLE
ocaml-variants: Improve build time by only calling make once

### PR DESCRIPTION
**Pull request by kit-ty-kate**
Proposal by @dra27 in https://github.com/ocaml/opam-repository/pull/17697

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
cc @Octachron, any thoughts?

File: packages/ocaml-variants/ocaml-variants.4.12.0+trunk/opam [hunk 1]
```diff
@@ -46,8 +46,10 @@ build: [
     "LIBS=-static" {ocaml-option-static:installed}
     "--disable-warn-error"
   ]
-  [make "-j%{jobs}%" {os != "cygwin"} "world"]
-  [make "-j%{jobs}%" {os != "cygwin"} "world.opt"] {!ocaml-option-bytecode-only:installed}
+  [make "-j%{jobs}%" {os != "cygwin"}
+    "world" {ocaml-option-bytecode-only:installed}
+    "world.opt" {!ocaml-option-bytecode-only:installed}
```
Comment (dra27) [inline packages/ocaml-variants/ocaml-variants.4.12.0+trunk/opam:51]:
```suggestion
  [make "-j%{jobs}%" {os != "cygwin"} "defaultentry"]
```
or just:
```suggestion
  [make "-j%{jobs}%" {os != "cygwin"}
```
will work too.

Comment (kit-ty-kate) [reply packages/ocaml-variants/ocaml-variants.4.12.0+trunk/opam]: ohh, thanks a lot, that's much better. Committed.

**Comment by kit-ty-kate** [comment]
Just for the archive, the latest commit is safe to do since https://github.com/ocaml/ocaml/commit/987b0814d7 (OCaml 4.10.0+beta1)

**Comment by Octachron** [comment]
This simplification sounds like a good idea.

**Pull request merged**